attack-theme/templates/general/two-column.html

```diff
@@ -4,7 +4,6 @@
 <div class="nav flex-column col-xl-2 col-lg-2 col-md-3  sidebar nav pt-5 pb-3 pl-3 border-right" id="v-tab" role="tablist" aria-orientation="vertical">
     <span class="heading" id="v-home-tab" aria-selected="false">MATRICES</span>
     <a class="nav-link side active show" id="v-attckmatrix-tab" data-toggle="tab" href="#v-attckmatrix" role="tab" aria-controls="v-attckmatrix" aria-selected="false">Enterprise</a>
-    <a class="nav-link side" id="v-privilegeescalation-tab" data-toggle="tab" href="#v-privilegeescalation" role="tab" aria-controls="v-privilegeescalation" aria-selected="false">Pre-attack</a>
```

Comment:
I'm not sure the `block heading` section of this file is used anymore. I think the expandable sidenav has completely replaced it. The only reference I can find is on the `tactic.html` template, but it doesn't appear to show up on the actual output. Doing a global search in output for `v-defenseevasion-tab` shows that this block never occurs in the output HTML. So I think we may be able to remove this block from the template entirely.

Reply:
That is to say, your removing pre-ATT&CK from here is fine, but we may just want to remove the entire block if it's essentially just dead code at this point.

Reply:
Yes, I can confirm that the `block heading` section was not being used anymore, so I have removed it.